code/datums/diseases/advance/advance.dm

```diff
@@ -424,4 +424,9 @@ var/list/advance_cures = 	list(
 		src << "<a href='?_src_=vars;Vars=\ref[D]'>[D.name] - [D.holder]</a>"
 */
 
+/datum/symptom/heal/proc/Heal(mob/living/M, datum/disease/advance/A)
+	var/get_damage = sqrt(20-"stage_rate")*(2+rand())

```

Comment:
I don't quite know to to test it @Giacom, my apologies. I'll try and setup a test virus using this code.

I'm trying to figure out how to find the total stage_speed to no luck.

Do you have any suggestions to getting the total stage_speed of a custom virus?